packages/console/src/pages/TenantSettings/TenantBasicSettings/ProfileForm/options.tsx

```diff
@@ -0,0 +1,23 @@
+import type { AdminConsoleKey } from '@logto/phrases';
+import { TenantTag } from '@logto/schemas';
+import type { ReactElement } from 'react';
+
+import type DangerousRaw from '@/components/DangerousRaw';
+
+export const tagOptions: Array<{
+  title: AdminConsoleKey | ReactElement<typeof DangerousRaw>;
+  value: TenantTag;
+}> = [
+  {
+    title: 'tenant_settings.profile.environment_tag_development',
+    value: TenantTag.Development,
+  },
+  {
+    title: 'tenant_settings.profile.environment_tag_staging',
+    value: TenantTag.Staging,
+  },
+  {
+    title: 'tenant_settings.profile.environment_tag_production',
+    value: TenantTag.Production,
+  },
+];
```

Comment:
No need to be a `tsx` file. And I think you can just put this options in the `index.tsx` file

Reply:
merged this to the `index.tsx` file (the place needs to use this variable).